From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Martijn Muijsers <martijnmuijsers@live.nl>
Date: Sun, 16 Jul 2023 13:01:20 +0200
Subject: [PATCH] Empty block and item test classes

Creates empty classes for unit tests related to the Block and Item classes.

License: Fiddle Public License Agreement 1.0 (included in license/FPL-1.0.txt)
Fiddle - https://fiddlemc.org

diff --git a/src/test/java/org/fiddlemc/fiddle/block/PerBlockTest.java b/src/test/java/org/fiddlemc/fiddle/block/PerBlockTest.java
new file mode 100644
index 0000000000000000000000000000000000000000..4ad77f8ce7b43b5b54d7f7d1f9dfdebdd0ca9d8f
--- /dev/null
+++ b/src/test/java/org/fiddlemc/fiddle/block/PerBlockTest.java
@@ -0,0 +1,15 @@
+// Fiddle - tests related to net.minecraft.world.level.block.Block
+
+package org.fiddlemc.fiddle.block;
+
+import net.minecraft.core.registries.BuiltInRegistries;
+import net.minecraft.world.level.block.Block;
+import org.bukkit.support.AbstractTestingBase;
+
+public class PerBlockTest extends AbstractTestingBase {
+
+    private Iterable<Block> getBlocks() {
+        return BuiltInRegistries.BLOCK;
+    }
+
+}
diff --git a/src/test/java/org/fiddlemc/fiddle/block/PerItemTest.java b/src/test/java/org/fiddlemc/fiddle/block/PerItemTest.java
new file mode 100644
index 0000000000000000000000000000000000000000..b6dcad9c6a26bbc2e06259b292f5f914b299c213
--- /dev/null
+++ b/src/test/java/org/fiddlemc/fiddle/block/PerItemTest.java
@@ -0,0 +1,15 @@
+// Fiddle - tests related to net.minecraft.world.item.Item
+
+package org.fiddlemc.fiddle.block;
+
+import net.minecraft.core.registries.BuiltInRegistries;
+import net.minecraft.world.item.Item;
+import org.bukkit.support.AbstractTestingBase;
+
+public class PerItemTest extends AbstractTestingBase {
+
+    private Iterable<Item> getItems() {
+        return BuiltInRegistries.ITEM;
+    }
+
+}
